packages/client/src/generation/TSClient/helpers.ts

```diff
@@ -81,8 +78,7 @@ export function getArgFieldJSDoc(
   const fieldName = typeof field === 'string' ? field : field.name
   if (JSDocs[action] && JSDocs[action]?.fields[fieldName]) {
     const singular = type.name
-    const plural = pluralize(type.name)
-    const comment = JSDocs[action]?.fields[fieldName](singular, plural)
+    const comment = JSDocs[action]?.fields[fieldName](singular)
```

Comment:
`singular` variable name now feels off since there's no `plural` counterpart anymore, maybe just
```suggestion
    const comment = JSDocs[action]?.fields[fieldName](type.name)
```
and remove the variable